src/Analyzers/Core/CodeFixes/NamingStyle/NamingStyleCodeFixProvider.cs

```diff
@@ -156,13 +156,14 @@ protected override async Task<IEnumerable<CodeActionOperation>> ComputePreviewOp
                 return SpecializedCollections.SingletonEnumerable(
                     new ApplyChangesOperation(await _createChangedSolutionAsync(cancellationToken).ConfigureAwait(false)));
             }
-            protected override async Task<IEnumerable<CodeActionOperation>> ComputeOperationsAsync(CancellationToken cancellationToken)
+
+            protected override async Task<ImmutableArray<CodeActionOperation>> ComputeOperationsAsync(IProgress<CodeAnalysisProgress> progress, CancellationToken cancellationToken)
```

Comment:
i moved existing subclasses over to overriding the new method.  that way if, in the future, they want to report progress (or want to call an API that reports progress) they will have the sink to pass along.